cpp/test/sg/umap_parametrizable_test.cu

```diff
@@ -258,27 +258,6 @@ class UMAPParametrizableTest : public ::testing::Test {
     if (!test_params.fit_transform) {
       return;
     }
-
-#if CUDART_VERSION >= 11020
-
-    if (!umap_params.multicore_implem) {
-      device_buffer<float> embeddings2(alloc, stream,
-                                       n_samples * umap_params.n_components);
-      float* e2 = embeddings2.data();
```

Comment:
I think it's a good idea to remove the `multicore_implem` argument but we should keep these assertions so we're testing reproducibility in both C++ and Python. We could probably use `ASSERT_ALMOST_EQUAL` though, instead of using a custom kernel .

Reply:
I will restore the test.

Reply:
Restored.